### PR TITLE
Updated the release notes to 2.2.1

### PR DESCRIPTION
**Pull request by NunoFilipeSantos**

### CODE REVIEW

File: timescaledb/overview/release-notes/index.md [hunk 1]
```diff
@@ -33,19 +32,33 @@ functionality works, and, as a result, made APIs simpler and more consistent,
...
-which simplifies the user journey if a user wants to change the schema and
-they have already compressed their hypertables. 
+Skip Scan optimization on single node and multinode, which significantly 
+improves the performance of queries with DISTINCT ON. This 
+optimization is not yet available for queries on distributed 
+hypertables.
+
+This release also adds a function to create a distributed 
+restore point, which allows performing a consistent restore of a 
+multi-node cluster from a backup.
```
Comment (mkindahl) [inline timescaledb/overview/release-notes/index.md:44]: Nit: We are in general writing too long sentences in our documentation. I prefer terse and distinct text.

```suggestion
This release also adds support for creating a distributed 
restore point. This allows performing a consistent restore of a 
multi-node cluster from a backup.
```

File: timescaledb/overview/release-notes/index.md [hunk 2]
```diff
@@ -33,19 +32,33 @@ functionality works, and, as a result, made APIs simpler and more consistent,
...
+
+The bug fixes in this release address issues with size and stats 
+functions, high memory usage in distributed inserts, slow distributed 
+ORDER BY queries, indexes involving INCLUDE, and single chunk query 
+planning.
```
Comment (mkindahl) [inline timescaledb/overview/release-notes/index.md:49]:
```suggestion
The bug fixes in this release address issues in size and stats 
functions, high memory usage in distributed inserts, slow distributed 
ORDER BY queries, indexes involving INCLUDE, single chunk query 
planning, compression, and policies.
```

File: timescaledb/overview/release-notes/index.md [hunk 3]
```diff
@@ -33,19 +32,33 @@ functionality works, and, as a result, made APIs simpler and more consistent,
...
+
+**PostgreSQL 11 deprecation announcement**
+
+Timescale is working hard on our next exciting features. To make that 
+possible, we require functionality that is unfortunately absent on 
+PostgreSQL 11. For this reason, we will continue supporting PostgreSQL 
+11 until mid-June 2021. Sooner to that time, we will announce the 
+specific version of TimescaleDB in which PostgreSQL 11 support will 
+not be included going forward.
```
Comment (mkindahl) [inline timescaledb/overview/release-notes/index.md:58]: Nit: Don't beat around the bush:
```suggestion
specific version of TimescaleDB in which PostgreSQL 11 support will be removed.
```

**Review by mkindahl — APPROVED** [review]
LGTM. I have a few suggestions for improvements, and some minor corrections.

**Review by ryanbooz — APPROVED** [review]

**Review by ryanbooz — CHANGES_REQUESTED** [review]
I'm going to just redo this. The changes were made off of `master` which wasn't updated yet... which was my fault for not clarifying. The rebase has just created too many things for the relative small change. My apologies.

**Pull request closed without merging**